package.json

```diff
@@ -220,9 +220,8 @@
     "yaml": "^1.10.0"
   },
   "scripts": {
-    "prepublish": "gulp post-npm-install",
+    "prepack": "gulp post-npm-install",
```

Comment:
Renamed to prepack according to my understanding of the big warning box in the documentation: https://yarnpkg.com/advanced/lifecycle-scripts#prepublish